content/lxd/third-party-integrations.md

```diff
@@ -41,6 +42,17 @@ Juju can be used to deploy a variety of workloads across many different clouds a
 Take a look at the Step-by-Step Guide for LXD in the [Juju Documentation](https://juju.is/docs/olm/lxd).
 
 
+## MAAS
+
+[<img src="/static/img/logo-maas.png" alt="MAAS logo"/ height="111" width="308" style="display:block;float:none;margin-left:auto;margin-right:auto;padding-top:1em;padding-bottom:1em;">](https://maas.io/)
+
+[MAAS](https://maas.io/) is an open-source server provisioning software tool for your data centre.
+
+Self-service, remote installation of Windows, CentOS, ESXi and Ubuntu on real servers turns your data centre into a bare metal cloud.
```

Comment:
I'd probably add a paragraph covering how it interacts with LXD, maybe something like:

```
MAAS integrates with LXD to provide easy creation of virtual machines. It can automatically deploy and configure LXD as part of the deployment of a physical machine or can be connected to an existing LXD deployment to dynamically create virtual machines on it.
```